docs/index.yaml

```diff
@@ -294,7 +294,17 @@ entries:
     version: 0.1.0
   secrets-provider:
   - apiVersion: v1
-    created: "2021-09-30T01:19:08.251425-07:00"
+    created: "2021-10-31T12:32:34.219133-04:00"
+    description: A Helm chart for deploying CyberArk Secrets Provider for Kubernetes
+    digest: a6801d0494ca59f79fa81b85d22f95f6082f2d9be73f3f0e8533b82bfcabb02d
+    home: https://github.com/cyberark/secrets-provider-for-k8s
+    icon: https://www.cyberark.com/wp-content/uploads/2015/12/cybr-aim.jpg
+    name: secrets-provider
+    urls:
+    - https://cyberark.github.io/helm-charts/secrets-provider-1.1.6.tgz
```

Comment:
Should this URL work yet?

Reply:
Not yet. The changes in this PR have to merge first.

Some background: The way that this works is that when this repo was first created, a [GitHub Pages site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site) was created for the repo. Now, anything merged into the `docs/` subdirectory, that also has an entry in the `docs/index.yaml` is publicly available at `https://cyberark.github.io/helm-charts/*`.

Unfortunately, this means we have to do a final test the link after the merge.